README.md

```diff
@@ -279,48 +335,59 @@ OR file_path = 'README.md';
 You can find further sample queries in the [examples](examples/README.md) folder.
 
 <!---
-#### 6. Next steps
+### 6. Next steps
 
-You can now run the source{d} Engine, choose what you would like to do next:
+You can now run source{d} Engine, choose what you would like to do next:
 
 - [**Analyze your git repositories**](#)
 - [**Understand how your code has evolved**](#)
 - [**Write your own static analysis rules**](#)
 - [**Build a data pipeline for MLonCode**](#)
 --->
 
+## Other Guides and Examples
+
+Here is a collection of documentation, guides, and examples of the components exposed by source{d} Engine:
+
+- [gitbase documentation](https://docs.sourced.tech/gitbase/): table schemas, syntax, functions, examples.
+- [Babelfish documentation](https://docs.sourced.tech/babelfish/): specifications, usage, examples.
 
 ## Architecture
 
-source{d} Engine functions as a CLI tool that provides easy access to components of the source{d} stack for Code As Data. It consists of a daemon managing all of the services (Babelfish, Enry, Gitbase etc.) which are packaged as docker containers.
+source{d} Engine functions as a command-line interface tool that provides easy access to components of source{d} stack for Code As Data.
+
+It consists of a daemon managing all of the services, which are packaged as Docker containers:
 
-<p align="center"><img src="docs/architecture-diagram.png" height="150" /></p>
+- [enry](https://github.com/src-d/enry): language classifier
+- [babelfish](https://doc.bblf.sh): universal code parser
+  - [daemon](https://github.com/bblfsh/bblfshd): Babelfish server
+  - [language drivers](https://github.com/search?q=topic%3Adriver+org%3Abblfsh&type=Repositories): parsers + normalizers for programming languages
+  - [babelfish-web](https://github.com/bblfsh/web): web client for Babelfish server
+- [gitbase](https://github.com/src-d/gitbase): SQL database interface to Git repositories
+  - [gitbase-web](https://github.com/src-d/gitbase-web): web client for gitbase
 
 For more details on the architecture of this project, read [docs/architecture.md](docs/architecture.md).
 
+<img src="./docs/architecture-diagram.png" alt="source{d} Engine"height="150" />
```

Comment:
Broken image, does not render.

Reply:
wow, fixed!

Reply:
I still see it does not render:
https://github.com/src-d/engine/blob/74e926e58cc1b2c1be5120b369a67d6a8e032bc0/README.md#architecture

Reply:
Removed HTML as agreed.
I double checked it, and it's rendering.